cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/distributed/AbstractProgramTwillRunnable.java

```diff
@@ -155,13 +157,21 @@ public void initialize(TwillContext context) {
       CommandLine cmdLine = parseArgs(context.getApplicationArguments());
 
       // Loads configurations
-      hConf = new Configuration();
+      hConf = HBaseConfiguration.create();
+      String numClientRetries = hConf.get(Constants.HBase.CLIENT_RETRIES);
```

Comment:
Something like `https://github.com/caskdata/cdap/blob/release/4.1/cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/distributed/AbstractDistributedProgramRunner.java#L188` (BTW, it is wrong to modify the injected `hConf` without copying).

The right thing to do is in the constructor of `AbstractDistributedProgramRunner`, make a copy of the hConf and then modify it in that class (like setting of the scheduler queue).